apps/modernization-ui/src/apps/page-builder/page/management/edit/subsection/Subsection.tsx

```diff
@@ -45,7 +48,14 @@ export const Subsection = ({ subsection, onAddQuestion }: Props) => {
                 fetch(page.id);
             });
         } else {
-            console.log('delete question NYI', id);
+            PageQuestionControllerService.deleteQuestionUsingDelete({
+                authorization: authorization(),
+                page: page.id,
+                questionId: Number(id)
+            }).then((resp: any) => {
```

Comment:
Issue: use of `any`. Looking at the controller, no response is expected unless there is an error so you should be able to just remove the `resp: any` all together. 
```typescript
.then(() => {
...
}
```